### ISSUE TITLE
[libx11] Failed to build

### ISSUE THREAD

**Issue by supechicken666** [skycocker/chromebrew#4641]
```shell
configure: error: Package requirements (xproto >= 7.0.25 xextproto xtrans xcb >= 1.11.1 kbproto inputproto) were not met:

No package 'xtrans' found

Alternatively, you may set the environment variables X11_CFLAGS
and X11_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
libx11 failed to install: `./configure --prefix=/usr/local --libdir=/usr/local/lib64 --mandir=/usr/local/share/man --build=x86_64-cros-linux-gnu --host=x86_64-cros-linux-gnu --target=x86_64-cros-linux-gnu` exited with 1
```

**Comment by uberhacker** [comment]
Missing libxtrans dependency?

**Comment by supechicken666** [comment]
Problem fixed by installing `libxtrans`, seems there are some problem with the `crew` dependency system since the `libxtrans` already a dependency package

**Comment by uberhacker** [comment]
Fixed in #4640.
